React/CoreModules/RCTAlertController.m

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#import <React/RCTUtils.h>
+
+#import "RCTAlertController.h"
+
+@interface RCTAlertController ()
+
+#if !TARGET_OS_OSX // [TODO(macOS ISS#2323203)
+@property (nonatomic, strong) UIWindow *alertWindow;
+
```

Comment:
While it shouldn't lead to issues here (other than perhaps a new warning?), it would be technically more correct to add a `#endif` here…